src/EnergyPlus/MixedAir.cc

```diff
@@ -3061,37 +3053,39 @@ namespace MixedAir {
 
 				// Make sure all zones with mechanical ventilation are on the correct air loop
 				TempMechVentArrayCounter = 0;
+				int AirLoopZoneInfoSize = static_cast <int> ( AirLoopZoneInfo.size() );
```

Comment:
That does not work. AirLoopNum is greater than zero (I think 2) and the size of AirLoopZoneInfo is only 1.